common/djangoapps/third_party_auth/saml.py

```diff
@@ -62,6 +62,30 @@ def _check_entitlements(self, idp, attributes):
                         "SAML user from IdP %s rejected due to missing eduPersonEntitlement %s", idp.name, expected)
                     raise AuthForbidden(self)
 
+    def _create_saml_auth(self, idp):
+        """ Get an instance of OneLogin_Saml2_Auth """
+        # We only override this method so that we can add extra debugging when debug_mode is True
+        # Note that auth_inst is instantiated just for the current HTTP request, then is destroyed
+        auth_inst = super(SAMLAuthBackend, self)._create_saml_auth(idp)
```

Comment:
Yeah, IdP is a pretty standard term used in SAML-lang ("IDentity Provider"). Since this is overriding an API method from an external repository (albeit one that I coded), I don't want to change the interface now. So what I've done is updated the docstring above to make it the parameter's meaning obvious.